Src/VimCore/CoreInterfaces.fs

```diff
@@ -2736,6 +2736,32 @@ with
         | Some transaction -> transaction.Complete()
         | Option.None -> ()
 
+[<RequireQualifiedAccess>]
+type SwitchCommandModeArgument =

```

Comment:
Why create a new type here instead of just expanding the values of `ModeArgument`?

Reply:
I made a correction, but I am not confident.
Because this PR is not urgent,
Please give priority to the release work of 2.8.0.